python/.coveragerc

```diff
@@ -1,5 +1,21 @@
 # Configuration file for Python coverage tests
 [run]
-include = cuml/*
 omit = cuml/test/*
-plugins = Cython.Coverage
\ No newline at end of file
+plugins = Cython.Coverage
+parallel = true
+source = cuml
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
+    if 0:
```

Comment:
Does this include the block that follows or just the `if 0: ` itself?

Reply:
This was removed.

Reply:
Yes, if the ignored line starts a new scope, the entire scope is ignored. Same applies to the `def __repr__` section.

Reply:
I added a link to the docs for clarity.